src/state.ts

```diff
@@ -28,6 +28,7 @@ class State {
         },
     ]
     last_ex_str = "echo"
+    lastFocusInputSelectors: Map<number, string> = new Map() // used for focusinput, not for changing tabs like prevInputs
```

Comment:
I can't think of a good reason why this should be separate from prevInputs. I'll probably merge the two.